CriteoGoogleAdapter/CriteoGoogleAdapter.xcodeproj/project.pbxproj

```diff
@@ -80,6 +83,7 @@
 		3CF2926D760C675680C3BA20 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				CF6AE25D24CEB22F00E00CDA /* CriteoPublisherSdk.framework */,
 				DFC3F4A323035EF500318644 /* CriteoPublisherSdk.framework */,
```

Comment:
FAR: This is probably a leftover of a previous version that should be removed

Reply:
Good catch ! I rebased to switch the two commits. And merging pbxproj is completely dumb :(

Reply:
Done